server.js

```diff
@@ -75,6 +75,7 @@ module.exports = function() {
         validate: {
           payload: {
             locale: Joi.string().min(2).max(12).required(),
+            newsletter: Joi.string().required(),
```

Comment:
Lets add a default of `'mozilla-foundation'` here with https://github.com/hapijs/joi/blob/master/API.md#anydefaultvalue-description